instance/migrations/0048_appserver_refactor1.py

```diff
@@ -101,7 +100,6 @@ class Migration(migrations.Migration):
             options={
                 'verbose_name': 'Open edX Instance',
             },
-            bases=(instance.models.instance.Instance, models.Model),
```

Comment:
@pomegranited Yeah, migrations getting retconned by later commits. :)

I wasn't surprised about your change, since it's documented that you need to access historical models in data migrations, and it was simply a bug in the migration code you fixed.  This case here, however, is a more fundamental design problem in Django migrations.  The original code was actually correct according to the documentation, but it didn't work anyway.